onnxruntime/test/onnx/main.cc

```diff
@@ -538,6 +561,34 @@ int real_main(int argc, char* argv[], Ort::Env& env) {
   broken_tests.insert({"mlperf_ssd_resnet34_1200", "unknown error"});
   broken_tests.insert({"tf_inception_v1", "flaky test"}); //TODO: Investigate cause for flakiness
 #endif
+
+#ifdef USE_DML
+  if (enable_dml)
+  {
+    broken_tests.insert({"PixelShuffle", "Test requires 6D Reshape, which isn't supported by DirectML"});
```

Comment:
do we plan to fix these tests before 1.0 or after?

Reply:
Hopefully before, but it depends on time. The vast majority of tests are passing and we have our own separate validation via WinML, so I don't consider these failures to be blocking. I suspect we'll simply fix as many as we can in the time we have left.